tests/utils.py

```diff
@@ -74,3 +77,23 @@ def wait_for_server(host, port, timeout=10):
     raise TimeoutError
 
 
+def configure_test_logging(argv):
+  # parse arguments but only handle '-v': argv may contain 
+  # other things meant for unittest argument parser
+  parser = argparse.ArgumentParser(add_help=False)
+  parser.add_argument('-v', '--verbose', action='count', default=0)
+  args, _ = parser.parse_known_args(argv)
+  
+  if args.verbose <= 1:
```

Comment:
In your commit message, you have mentioned that the number of `v` means the following:
```
default: ERROR
"-v":    ERROR, but unittest prints test names
"-vv":   WARNING
"-vvv":  INFO
"-vvvv": DEBUG
```

I think this should be documented somewhere.
Can we have like `--help` argument and functionality?

Reply:
I wish we could... but I don't see a way to do that without breaking unittest --help. The lack of discoverability is annoying but I just don't know how to make it better (without essentially hiding unittest functionality, which I also don't want to do)